Maquetacion/scss/variables.scss

```diff
@@ -0,0 +1,14 @@
+$font-page: 'Roboto', sans-serif;
```

Comment:
you can splitt this variables by it's ussage instead of having all in the same file